lib/explorer/shared.ex

```diff
@@ -2,7 +2,7 @@ defmodule Explorer.Shared do
   # A collection of **private** helpers shared in Explorer.
   @moduledoc false
 
-  @non_list_types [
+  @scalar_types [
```

Comment:
With struct type being added the name didn't seem right anymore. Hope I got the intention right here. 😅